Includes/Core/Geometry/BoundingBox.hpp

```diff
@@ -47,9 +47,6 @@ class BoundingBox
 {
  public:
     static_assert(N > 0, "Dimension should be greater than 0");
-    static_assert(
-        std::is_floating_point<T>::value,
```

Comment:
Could I ask you why floating point type assertion was deleted? If given template type N is integer type, below method doesn't work properly. 
```c++
template <typename T, size_t N>
T BoundingBox<T, N>::DiagonalLength() const
{
    return VectorType(upperCorner - lowerCorner).Length();
}
```

Reply:
Oh, I'll fix it tonight. Thanks. 🕊️

Reply:
I restored 'static_assert'. 😄